contents/swift/welcome-to-swift.md

```diff
@@ -82,9 +84,9 @@ func yearAlbumReleased(name: String) -> Int? {
 }
 ```
 
-### Optional Binding
+#### Optional Binding
 
-And here is how you unwrap the Optional safely using an `if-let` statement:
+Then, you can unwrap the Optional safely using an `if-let` statement, and handle the special value separately:
```

Comment:
It's not immediately clear what you mean by special value... are you referring to the value present in the optional or the `nil` value?

Perhaps you could change this to:

> ..., and handle the value or `nil` separately.